src/Shared/FileUtilities.cs

```diff
@@ -401,8 +401,10 @@ internal static string FixFilePath(string path)
         /// first segment exists and is a directory.
         /// Use a native shared method to massage file path. If the file is adjusted,
         /// that qualifies is as a path.
+        ///
+        /// @baseDirectory is just passed to LooksLikeUnixFilePath, to help with the check
         /// </summary>
-        internal static string MaybeAdjustFilePath(string value)
+        internal static string MaybeAdjustFilePath(string value, string baseDirectory="")
```

Comment:
I had tried that first and got `error CS1736: Default parameter value for 'baseDirectory' must be a compile-time constant`.